spec/domain/helpers/permissions_spec.exs

```diff
@@ -0,0 +1,272 @@
+defmodule Squadster.Domain.Helpers.PermissionsSpec do
+  use ESpec.Phoenix, async: true
+
+  import Squadster.Support.Factory
+
+  alias Squadster.Helpers.Permissions
+  alias Squadster.Repo
+
+  let :user do
+    member = insert(:squad_member, role: :commander)
+    insert(:user, squad_member: member)
+  end
+
+  let :squad do
+    %{squad_member: %{squad: squad}} = user() |> Repo.preload(squad_member: :squad)
+    squad
+  end
+
+  let :another_squad do
```

Comment:
Probably, this name is not so clear as it necessary
Can we change it to `owned_squad` and `not_owned_squad` or something like that.
Please, share your opinion

Reply:
maybe not `owned`, e.g. journalist not own the squad